server/expression.js

```diff
@@ -199,14 +259,14 @@ expression.listMatrix_ = function(expressionPath) {
 };
 
 /**
- * Reads expression matrix from text file.
- * @param {string} expressionFile Path to the expression file.
+ * Reads matrix from text file.
```

Comment:
Write in comments to indicate that this reads a sub-matrix selected by geneNames and conditionNames.